Action Cable Guide - add Transmit method (#51451)
rails/rails: guides/source/action_cable_overview.md

client that sent the message. Note that params are the same as they were when
you subscribed to the channel.

### Transmiting To A Subscriber

> In some scenarios you may want to send data to just the subscriber that initiated a subscription request, or that sent a message.

Reviewer: ```suggestion
In certain scenarios, there may be a need to send data exclusively to the subscriber who initiated a subscription request or sent a message.
```

> In these cases we can make use of [`transmit`][] to send a hash of data to the subscriber. The hash will automatically be wrapped in a JSON envelope with the proper channel identifier marked as the recipient.

Reviewer: ```suggestion
In these cases, [`transmit`][] can be utilized to send a hash of data to the subscriber. The hash will automatically be encapsulated in a JSON envelope with the appropriate channel identifier designated as the recipient.
```
